src/porepy/utils/interpolation_tables.py

```diff
@@ -20,11 +20,15 @@
 constant approximations of derivatives.
 
 """
+from __future__ import annotations
+
 import itertools
-from typing import Callable, Iterator, List, Tuple
+from typing import Callable, Iterator, Optional
 
 import numpy as np
 
+import porepy as pp
+
 
 class InterpolationTable:
```

Comment:
The construction for `AdaptiveInterpolationTable` is natural and it reads :
`pp.AdaptiveInterpolationTable(dx, base_point, function, dim)`

Maybe `InterpolationTable` should mirror the above constructor instead of
`pp.InterpolationTable(low, high, npt, function, dim)`

Reply:
The `InterpolationTable` will need to know its upper bound, so there cannot be a 1-1 correspondence. I prefer giving the upper bound and the number of points explicitly.